behave/features/cla_public/public_exit_page.feature

```diff
@@ -0,0 +1,34 @@
+@cla_public
+Feature:
+  - Testing the "Exit this page" journey through all possibilities available for a user
+
+  Background: Start page
+    Given I have selected the start now button on the start page
+    And I am taken to the "Choose the area you most need help with" page located on "/scope/diagnosis/"
+
+  @exit-behaviour-button
+  Scenario: Describes how to exit from the domestic abuse section and the website safely when clicking the exit button
+    When I select the category <category>
+      | category       |
+      | Domestic abuse |
+    And I am taken to the "Choose the option that best describes your personal situation" page located on "/scope/diagnosis/n43n3"
+    Then The "Exit this page" button is on the page and I click it
+    And I am diverted to the BBC website
```

Comment:
Very sorry I didn't spot this before, but the end of your Scenarios don't follow the Gherkin Keywords best standards.

Thankfully this is super easy and a quick fix.
https://www.browserstack.com/guide/cucumber-best-practices-for-testing

Given When, And, Then.  The generally rule of thumb is to always start with a Given and end with a Then. I've attached a good article that coverers the reasoning why.

The Given is being used at the start already, which is in the background scenario, so that's fine, but we want them scenarios to end with `Then`

Reply:
An example on this one would be: 

 When
 And
 And 
 Then